QUARKUS-6259.md

```diff
@@ -0,0 +1,38 @@
+# QUARKUS-6245 Enable named data sources for Hibernate Reactive
+
+JIRA:  https://issues.redhat.com/browse/QUARKUS-6259
+
+Upstream PRs:
+    - https://github.com/quarkusio/quarkus/pull/47631
+    - https://github.com/quarkusio/quarkus/pull/49287
+
+Documentation:
+    - https://quarkus.io/version/main/guides/hibernate-reactive#multiple-persistence-units
+
+This feature allow to configure multiple (named) datasources with hibernate-reactive extension.
+By extend it allow named persistence units.
+
+## Scope of testing
+Tests will cover basic working with an app, that is connected to multiple databases.
+There will be 3 databases used in total - two of same type (e.g. postgres) and one different. 
+Goal is to verify if Quarkus with hibernate-reactive correctly handles multiple types of DB at once and if same driver can work with multiple DBs simultaneously.
+As part of this testing, every DB will be initiated with separate sql-load-script to verify this works properly.
+
+## Existing test coverage
+Upstream tests were added as part of https://github.com/quarkusio/quarkus/pull/47631, but these cover only work with one named datasource.
+Which is not sufficient, as we can have multiple datasources.
+Our testsuite already has a significant coverage for hibernate in module `sql-db/hibernate-reactive`.
+If possible tests for this feature should reuse this test coverage, only change underlying app to use multiple named datasources.
+
+## Automated test development
+New test class will be added into module `sql-db/hibernate-reactive`, 
+if this it is possible to implement named datasources there, without disrupting existing tests.
+If named datasources prove to be too disruptive, new module will be created.
```

Comment:
```suggestion
named datasources should be implemented there, unless they disrupt existing tests.
Otherwise, a new module needs to be created.
```

Reply:
I omitted this entire sentenses, so closing this one.